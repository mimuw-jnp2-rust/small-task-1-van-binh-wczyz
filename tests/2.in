Jan Kowalski

n
n
Jan Kowalski
tofu
tofu
chicken
tofu
kotlet schabowy
mleko

n
y
Jan Kowalski
n
tofu

y
