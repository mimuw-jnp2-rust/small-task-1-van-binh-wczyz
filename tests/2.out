Hi! Welcome to Van Binh! What's your name?
Welcome, Jan Kowalski!
Enter dish name or empty line to finish:
Takeaway? (y/n)
Would you like to save this order? (y/n)
Your order is empty!
Hi! Welcome to Van Binh! What's your name?
Welcome, Jan Kowalski!
Enter dish name or empty line to finish:
Enter dish name or empty line to finish:
Enter dish name or empty line to finish:
Enter dish name or empty line to finish:
Enter dish name or empty line to finish:
Unknown dish name: kotlet schabowy
Enter dish name or empty line to finish:
Unknown dish name: mleko
Enter dish name or empty line to finish:
Takeaway? (y/n)
Would you like to save this order? (y/n)
This is order no. 1
There you go: chicken: 1, tofu: 3, rice: 0, takeway: false, it's going to be 65 zł
Hi! Welcome to Van Binh! What's your name?
Welcome back, Jan Kowalski!
Same as usual? (y/n)
Enter dish name or empty line to finish:
Enter dish name or empty line to finish:
Takeaway? (y/n)
This is order no. 2
There you go: chicken: 0, tofu: 1, rice: 0, takeway: true, it's going to be 16 zł
Hi! Welcome to Van Binh! What's your name?
Bye!
